.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,7 @@
+<!-- Please enter the corresponding issue ID: -->
 **Fixes: #Issue**
```

Comment:
`Fixes` seems only applicable to bugs. Maybe use `Closes` as a more general term? (also supported by GitHub)
```suggestion
**Closes: #Issue**
```